File: Assignment7/program5.c
#include<stdio.h>

 void Display_Table(int iNo)
 {
    int iCnt;
    int mul = 1;

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = 1 ;iCnt <= 10 ; iCnt++) 
    {
         mul = iCnt * iNo;
        {
            printf("%d\n",mul);
        }
    }
 }

int main()
{
  int iValue = 0;

  printf("Enter the Number: ");
  scanf("%d",&iValue);

  Display_Table(iValue);

    return 0;
}

File: Assignment21/program5.java

// 5. Write a program to print all numbers from 1 to N that are divisible by both 2 and 3.

package Assignment21;

import java.util.Scanner;

class program5 
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the Number : ");
        iValue = sobj.nextInt();

        Divisible dobj = new Divisible();
        dobj.Is_Divisible(iValue);

        sobj.close();
    }
}

class Divisible
{
    public void Is_Divisible(int iValue)
    {
        int iCnt = 0;

        System.out.println("Numbers from 1 to N that are divisible by both 2 and 3 are : ");
        for(iCnt = 1; iCnt <= iValue; iCnt++)
        {
            if(iCnt % 2 == 0 && iCnt % 3 == 0)
            {
                System.out.println(iCnt);
            }
        }
    }

}

